<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>196</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Atkinson Hyperlegible</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="main_layout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="left_layout">
        <item>
         <widget class="QRadioButton" name="shutdown_radioButton">
          <property name="font">
           <font>
            <family>Atkinson Hyperlegible</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Shutdown</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="hibernate_radioButton">
          <property name="font">
           <font>
            <family>Atkinson Hyperlegible</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Hibernate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="sleep_radioButton">
          <property name="font">
           <font>
            <family>Atkinson Hyperlegible</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Sleep</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="right_layout">
        <item>
         <widget class="QTimeEdit" name="timeEdit">
          <property name="font">
           <font>
            <family>Atkinson Hyperlegible</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="currentSection">
           <enum>QDateTimeEdit::HourSection</enum>
          </property>
          <property name="displayFormat">
           <string>HH : mm : ss</string>
          </property>
          <property name="time">
           <time>
            <hour>1</hour>
            <minute>0</minute>
            <second>0</second>
           </time>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="button_layout">
          <item>
           <widget class="QPushButton" name="start_button">
            <property name="font">
             <font>
              <family>Atkinson Hyperlegible</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pause_button">
            <property name="font">
             <font>
              <family>Atkinson Hyperlegible</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Pause</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stop_button">
            <property name="font">
             <font>
              <family>Atkinson Hyperlegible</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="donow_button">
          <property name="font">
           <font>
            <family>Atkinson Hyperlegible</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Do now</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>473</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSet_default_style"/>
    <addaction name="actionSet_dark_style"/>
   </widget>
   <addaction name="menuView"/>
  </widget>
  <action name="actionSet_default_style">
   <property name="text">
    <string>Set Default Style</string>
   </property>
  </action>
  <action name="actionSet_dark_style">
   <property name="text">
    <string>Set Dark Style</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
